Automattic/wp-calypso
# Reader: sort 'Followed Sites' by latest update, not alphabetical

* Requires a wpcom patch: D67552-code to provide information from the backend


#### Changes proposed in this Pull Request

* Followed Sites in the reader are now sorted by last updated order

#### Testing instructions



* Apply D67552-code to sandbox
* Sandbox public-api
* Make sure you are following some wordpress.com blogs
* Make sure you are following some RSS feeds
  * Visit https://wordpress.com/following/manage
    * Add `http://wirecutter.com`
    * Add `http://www.loc.gov/rss/pao/news.xml`
    * Add `http://feeds.feedburner.com/euronews/en/home/`
* Visit http://calypso.localhost:3000/read
* Expand "followed sites" on the left hand sidebar
* They should be in most recently updated order, with the most recently updated sites on the top
![2021-09-29_13-03](https://user-images.githubusercontent.com/937354/135324202-0ca67120-fbdb-4c4b-ad09-50b2b67f5c39.png)




Related to #43592
